### PR TITLE
Change wall exporting to .wall instead of .pixie

### PR DESCRIPTION
**Pull request by raineaeternal**
Pretty much what the title says.

### CODE REVIEW

File: UnityProject/Assets/Editor/CompileMaterialWindow.cs [hunk 1]
```diff
@@ -30,7 +30,7 @@ private void OnGUI()
             material.Icon = (Texture2D)EditorGUILayout.ObjectField("Cover Image", material.Icon, typeof(Texture2D), false);
 
             bool disableExportButton = false;
-            if (material.transform.Find("Pixie") == null
+            if (material.transform.Find("Wall") == null
```
Comment (Pespiri) [inline UnityProject/Assets/Editor/CompileMaterialWindow.cs]: This should not be changed as it is the actual name of gameObject in the Unity editor and is not related to the filename/file extension that you intend to change.

**Review by Pespiri — CHANGES_REQUESTED** [review]

File: UnityProject/Assets/Editor/CompileMaterialWindow.cs (hunk 1 above)
Comment (raineaeternal) [reply UnityProject/Assets/Editor/CompileMaterialWindow.cs]: Will revert.

**Comment by raineaeternal** [comment]
Reverted changes.

File: UnityProject/Assets/Editor/CompileMaterialWindow.cs [hunk 2]
```diff
@@ -17,7 +17,7 @@ public static void ShowWindow()
 
     private void OnGUI()
     {
-        GUILayout.Label("Walls", EditorStyles.boldLabel);
+        GUILayout.Label("Pixie", EditorStyles.boldLabel);
```
Comment (Pespiri) [inline UnityProject/Assets/Editor/CompileMaterialWindow.cs]: This should remain "Walls" as it's just the header in the Unity editor menu. You probably meant to change line 33 back

**Review by Pespiri — CHANGES_REQUESTED** [review]

File: UnityProject/Assets/Editor/CompileMaterialWindow.cs (hunk 2 above)
Comment (raineaeternal) [reply UnityProject/Assets/Editor/CompileMaterialWindow.cs]: My b, whoops-

**Comment by raineaeternal** [comment]
Fixed said issue.

**Pull request merged**